### DISCUSSION TITLE
[Errno 2] No such file or directory: '..../MPRAGEMETA.csv'

### DISCUSSION THREAD

**Discussion by hnese** [aramis-lab/clinica#1068] [Q&A]
I have tried to convert one subject's fMRI and T1 data from ADNI3 advanced dataset. I have downloaded clinical data (csv files) as stated in the instructions. However, I got the error 
[Errno 2] No such file or directory: '/.../ADNI_clinical/MPRAGEMETA.csv'

When I checked I couldn't find the file 'MPRAGEMETA.csv' indeed. I couldn't find it in ADNI page either. 
I hope you can help.

Thank you in advance,
Hüden

**Comment by NicolasGensollen** [comment]
Hi @hnese 

You should definitely have this file in the clinical data you downloaded. If you downloaded the data recently, you should have the date at the end of the file name (something like "MPRAGEMETA_30Jan2024.csv").

Could you provide more information about your setup (OS name and version, Python version, Clinica version, the command the ran in the terminal, the full error trace that you get...)?

**Comment by hnese** [comment]
I see, found it! I have deleted the date at the end of the files and it worked.

Thank you very much for the quick reply.

Best,
Hüden

**Reply by NicolasGensollen** [reply]
@hnese Great !
Note that I still think you should update your version of Clinica to 0.7.7 in which we added support for the new convention used by ADNI (dates at the end of CSV files). I suspect you will encounter issues at other places otherwise.

**Reply by xupin262** [reply]
> 明白了，找到了！我删除了文件的日期，现在好了。非常感谢您的快速回复。祝好，Hüden
> […](#)
> 2024年2月6日 13:19，Gensollen ***@***.***> 写道：您好@hnese < https://github.com/hnese > 您下载的临床数据中肯定应该有这个文件。如果最近下载了这些数据，文件名会包含日期（例如“MPRAGEMETA_30Jan2024.csv”）。您可以提供更多有关您设置的信息（例如名称和版本、Python版本）网络、Clinica版本、您在终端中运行的命令、您收到的完整错误跟踪信息……）？ — 直接回复此邮件，在 GitHub 上查看 < [#1068 (评论)](https://github.com/aramis-lab/clinica/discussions/1068#discussioncomment-8380915) >，或取消订阅 < https://github.com/notifications/unsubscribe-auth/AGGASZHE6VAJ2KV4QJ2ODR3YSH7UJAVCNFSM6AAAAABCY4W4E2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DGOBQHEYTK >。您收到此邮件是因为您在 GitHub 上已被撤消。

Hi,

I meet the same issue. ADNI doesn't provide MPRAGEMETA.csv (MRIQUALITY.csv 和 MAYOADIRL_MRI_IMAGEQC_12_08_15.csv) any more after October 2024. Can you happened to download those file share it? I appreciate it.

Best,
Pin Xu
